docs/src/main/sphinx/release/release-400.md

```diff
@@ -0,0 +1,52 @@
+# Release 400 (12 Oct 2022)
+
+## General
+
+* Add output buffer utilization to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14396`)
+* Increase concurrency for large clusters. ({issue}`14395`)
+
+## BigQuery connector
+
+* Add support for [truncating tables](/sql/truncate). ({issue}`14494`)
+
+## Delta Lake connector
+
+* Prevent coordinator out-of-memory failure when querying a large number of
+  tables in a short period of time. ({issue}`14571`)
+
+## Hive connector
+
+* Reduce memory usage when scanning a large number of partitions, and add the
+  `hive.max-partitions-for-eager-load` configuration property to manage the
+  number of partitions that can be loaded into memory. ({issue}`14225`)
+* Increase the default value of the `hive.max-partitions-per-scan`
+  configuration property to 1,000,000 from 100,000. ({issue}`14225`)
```

Comment:
I know that formatting makes it more readable .. this sort of format wont work in the actual property I think .. so should we leave it or change to code highlighting like so

```
`1000000` from `100000`
```